tools/js2c.py

```diff
@@ -262,6 +219,45 @@ def get_js_contents(js_path, is_debug_mode=False):
     return code
 
 
+def get_literals_from_snapshots(snapshot_tool, snapshot_list):
+    literals_path = fs.join(path.SRC_ROOT, 'js','literals.list')
```

Comment:
All of the generated  temporary files are in the `src` directory at the moment (`*.js.snapshot`, `*.js.wrapped`). It would be a good idea to move them into build directory, however it is quite unrelated to this change. I'd do it in a separate patch.

Reply:
That's ok for me.